wemod_launcher/pyinstaller.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+from pathlib import Path
+
+HERE = Path(__file__).parent.absolute()
+PATH_TO_MAIN = str(HERE / "cli.py")
+
+
+def install():
+    args = [
+        "--onefile",
+        "--windowed",
+        "--clean",
+        "--hidden-import tkinter",
+        "--collect-all tkinter",
+        PATH_TO_MAIN,
+    ]
+    subprocess.run("poetry run pyinstaller " + " ".join(args), shell=True)
```

Comment:
May want to use the argument list directly, i think that is a bit nicer and ensures that the arguments are escaped correctly, so like: 
process = subprocess.Popen(arglist, stdout=subprocess.PIPE)
that supports lists and will treat each list item as one argument.